src/pybind/mgr/dashboard/tests/test_rbd_mirroring.py

```diff
@@ -10,8 +10,10 @@
     import unittest.mock as mock
 
 from .. import mgr
+from ..controllers.orchestrator import Orchestrator
 from ..controllers.rbd_mirroring import RbdMirroring, \
-    RbdMirroringPoolBootstrap, RbdMirroringSummary, get_daemons, get_pools
+    RbdMirroringPoolBootstrap, RbdMirroringStatus, RbdMirroringSummary, \
+    get_daemons, get_pools
```

Comment:
@nizamial09 backporting this I saw that `get_daemons get_pools` are not used

Reply:
i think in master its used. maybe not in the backported branch? @pereman2